receiver/jaegerreceiver/config.go

```diff
@@ -125,6 +127,10 @@ func (cfg *Config) Validate() error {
 		if len(cfg.RemoteSampling.StrategyFile) != 0 && grpcPort == 0 {
 			return fmt.Errorf("strategy file requires the gRPC protocol to be enabled")
 		}
+
+		if cfg.RemoteSampling.StrategyFileReloadInterval < 0 {
```

Comment:
If I don't specify a value at all, won't it hit this condition?

Reply:
not hit, default it is zero, it is valid. just means not auto reload.